nomos-core/src/da/certificate/mock.rs

```diff
@@ -0,0 +1,103 @@
+use std::collections::HashMap;
+
+use crate::da::{attestation::Attestation, certificate::Certificate};
+
+#[derive(Clone, Debug, PartialEq)]
+pub struct MockAttestation {
+    voter: [u8; 32],
+    signature: Vec<u8>,
+}
+
+impl MockAttestation {
+    pub fn new(voter: &[u8; 32], signature: &[u8]) -> Self {
+        MockAttestation {
+            voter: *voter,
+            signature: signature.to_vec(),
+        }
+    }
+}
```

Comment:
As well probably feature gated? Or do we plan to remove this once we have the actual stuff in place?